src/odb/test/cpp/TestLef58Properties.cpp

```diff
@@ -54,10 +54,9 @@ BOOST_AUTO_TEST_CASE(test_default)
 
   lefParser.createTechAndLib("tech", libname, path.c_str());
 
-  FILE* write;
   path = std::string(std::getenv("BASE_DIR"))
          + "/results/TestLef58PropertiesDbRW";
-  write = fopen(path.c_str(), "w");
+  std::ofstream write(path, std::ios::binary);
```

Comment:
add exceptions, reduces questions upon search the code for lack of io exceptions...